80s Game/Assets/Scripts/UI Scripts/ScoreBehavior.cs

```diff
@@ -80,6 +81,20 @@ void Update()
             {
                 highScoreTextObject.SetText(highScoreText + highScore);
             }
+
+            //Update bottom score of leaderboard (edge case of null initials)
+            if (PlayerData.activePlayers[0].initials == null)
+            {
+                bottomScoreTextObject.SetText("AAA\t" + score);
+                bottomScoreTextObject.gameObject.transform.GetChild(0).GetComponent<TextMeshProUGUI>().text = bottomScoreTextObject.text;
```

Comment:
this seems to repeat it - consider moving this out of the if/else statements and instead set the value of what text is being saved within the if/else statements. That way we don't repeat that much code.